lib/cli.js

```diff
@@ -9,8 +9,19 @@ const configuration = getConfig();
 const choicesList = choices(configuration);
 const questionsList = questions(choicesList);
 
-inquirer.prompt(questionsList).then((answers) => {
-  const message = answers.moreInfo ? `${answers.editor}` : `${answers.type} ${answers.description}`;
+const prompt = inquirer.prompt(questionsList);
+
+prompt.ui.process.subscribe((answer) => {
+  if (answer.name === 'commitMessageWarning') {
+    const continueExecution = answer.answer;
+    if (!continueExecution) {
+      process.exit();
```

Comment:
I think the best solution would be, that we have rules in the `.sgcrc` file and if a rule is true than it should be enforced.  When writing a rule, why should we only show a warning? A rule is written to stick to it and not that anybody can break it.